mlflow/R/mlflow/R/python.R

```diff
@@ -1,6 +1,10 @@
 # Computes path to Python executable within conda environment created for the MLflow R package
 #' @importFrom reticulate conda_list
 get_python_bin <- function() {
+  in_env <- Sys.getenv("MLFLOW_PYTHON_BIN")
+  if (file.exists(in_env)) {
+    return(in_env)
```

Comment:
If the environment variable is set we should always use it. If the file does not exist we should throw an error instead of falling back to behavior without the env var.